docs/core-concepts/app-deployment.md

```diff
@@ -1,31 +1,283 @@
+
+1. Create a directory for your project, and from the command line, type `dotnet new` to create a new C# console project.
+
+2. Open the `Program.cs` file in an editor, and replace the auto-generated code with the following code. It prompts the user to enter text, and then displays the individual words entered by the user. It uses the regular expression `\w+` to separate the words in the input text.
+
+    ```cs
+    using System;
+    using System.Text.RegularExpressions;
+
+    namespace Applications.ConsoleApps
+    {
+        public class ConsoleParser
+        {
+            public static void Main()
+            {
+                 Console.WriteLine("Enter any text, followed by <Enter>:\n");
+                 String s = Console.ReadLine();
+                 ShowWords(s);
+                 Console.Write("\nPress any key to continue... ");
+                 Console.ReadKey();
+          }
+
+          private static void ShowWords(String s)
+          {
+              String pattern = @"\w+";
+              var matches = Regex.Matches(s, pattern);
+              if (matches.Count == 0)
+                  Console.WriteLine("\nNo words were identified in your input.");
+              else {
+                  Console.WriteLine("\nThere are {0} words in your string:", matches.Count);
+                  for (int ctr = 0; ctr < matches.Count; ctr++)
+                      Console.WriteLine("   #{0,2}: '{1}' at position {2}", ctr,
+                                        matches[ctr].Value, matches[ctr].Index);
+              }
+          } 
+      }
+    }
+    ```
+
+3. Open the `project.json` file and find the `dependencies` section, which should appear as follows:
+
+    "dependencies": {
+       "Microsoft.NETCore.App": {
+         "type": "platform",
+         "version": "1.0.0
+       }
+    },
+
+ Remove the following line in in the `dependencies` section:
+
+    ```project.json
+    "type": "platform",
+    ```
+   The presence of `"type": "platform"` identifies `Microsoft.NETCore.App` as a platform package that is provided by the system.
+
+4. Create a `runtimes` section in your `project.json` file that defines the platforms your app targets and specify the runtime identifier of each platform that you target. See [Runtime IDentifier catalog](https://dotnet.github.io/docs/core-concepts/rid-catalog.html "Runtime IDentifier (RID) catalog") for a list of runtime identifiers. For example, the following `runtimes` section indicates that the app runs on 64-bit Windows 10 operating systems and the 64-bit OSX Version 10.10 operating system.
```

Comment:
OSX -> OS X (I think in this case it's ok to use OS X since you're talking about a specific version)